run_test_harness_server.sh

```diff
@@ -1,2 +1,8 @@
-PREFAB_LOG_CLIENT_BOOTSTRAP_LOG_LEVEL=debug PREFAB_CLOUD_HTTP=true PREFAB_GRPC_URL="localhost:50051" ruby -Ilib test/harness_server.rb
-#GRPC_TRACE=all GRPC_VERBOSITY=DEBUG PREFAB_LOG_CLIENT_BOOTSTRAP_LOG_LEVEL=debug PREFAB_CLOUD_HTTP=true PREFAB_API_KEY="1|local_development_api_key" PREFAB_API_URL="localhost:50051" ruby -Ilib test/harness_server.rb
+#! /usr/bin/env bash
+
+PREFAB_CDN_URL="https://api-prefab-cloud.global.ssl.fastly.net" \
```

Comment:
these deserve a think at some point.

Reply:
Agreed. This commit really just satisfies the MVP to get things running.